src/tools/log-parser-rs/README.md

```diff
@@ -0,0 +1,79 @@
+# `kata-log-parser`
+
+## Introduction
+
+`kata-log-parser` is a tool that combines logfiles generated by the various
+system components, sorts them by timestamp, and re-displays the log entries. A
+time delta is added to show how much time has elapsed between each log entry.
+
+The tool is also able to check the validity of all log records, can re-format the
+logs, and output them in a different format.
+
+For more information on the `kata-log-parser` tool, use the help command:
+
```

Comment:
I see that `InputFormat` isn't implemented yet, but presumably the default will be rust runtime format logfiles. Either way, could you add an [admonition](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#warnings-and-other-admonitions) here stating what the default is and how to switch to the other format.

Related to this, we need to deal with a complication of the user knowing which runtime they are running. The current way to tell is:

```bash
$ containerd-shim-kata-v2 --version|grep -qi rust && echo rust || echo golang
rust
```

However, I've raised an issue to [add that feature to `kata-ctl'](https://github.com/kata-containers/kata-containers/issues/6518).

Also, it would be useful to tell users how to extract the log records from the system journal, including a reference to [enabling debug for containerd](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#enabling-full-containerd-debug).

Reply:
To be clear, will be writing README before I move this PR out of draft.